framework/src/application/node/node.js

```diff
@@ -241,6 +241,24 @@ module.exports = class Node {
 					action.params.password,
 					action.params.forging,
 				),
+			getAccount: async action =>
+				this.chain.dataAccess.getAccountsByAddress([action.params.address]),
+			getAccounts: async action =>
+				this.chain.dataAccess.getAccountsByAddress(action.params.address),
+			getBlockByID: async action =>
```

Comment:
Also, in general, these has to be serialized when it goes out of node context

Reply:
It's still within the context of the node. Do you mean for custom module context?

Reply:
yes when the data goes out from channel or network, it always should be serialized